specification/web/resource-manager/Microsoft.Web/stable/2024-04-01/CommonDefinitions.json

```diff
@@ -3112,6 +3112,20 @@
               "description": "Identity to use for Key Vault Reference authentication.",
               "type": "string"
             },
+            "AutoGeneratedDomainNameLabelScope": {
```

Comment:
Name should be CamelCase and can we have more meaningful name of the property please

Reply:
autoGeneratedDomainNameLabelScope is defined by DNL team and cannot be changed by RPs.